packages/nitro-protocol/src/contract/outcome.ts

```diff
@@ -29,26 +30,25 @@ export function decodeGuarantee(encodedGuarantee: Bytes): Guarantee {
 }
 
 export function isGuarantee(
-  allocationOrGuarantee: Allocation | Guarantee
+  allocationOrGuarantee: NitroAllocation | Guarantee
 ): allocationOrGuarantee is Guarantee {
   return !isAllocation(allocationOrGuarantee);
 }
 
 // Allocation and functions
-export type Allocation = AllocationItem[];
-export interface AllocationItem {
-  destination: Bytes32;
-  amount: Uint256;
-}
 
-export function encodeAllocation(allocation: Allocation): Bytes32 {
+// Note that a NitroAllocation is as defined in Outcome.sol, but an Allocation, used in
+// other parts of the codebase refers to what is used in the client-api-schema JSON-RPC
+export type NitroAllocation = AllocationItem[];
```

Comment:
I'm not a huge fan of having a specially named `Allocation` since all our contracts and documentation refers to `Allocation`. However we can always rename it back so it's not a big deal.

Reply:
Yeah I think we need to decide on which is the true `Allocation`. Maybe the JSON-RPC parameter ought to be instead `AssetAllocations`?